internal/data/hosts/hosts.go

```diff
@@ -49,6 +56,38 @@ func (r *hostsRepo) Save(ctx context.Context, model *biz.Host) (*biz.Host, error
 			return nil, err
 		}
 	}
+
+	if r.Authz != nil {
+		rels := []*kessel.Relationship{{
+			Resource: &kessel.ObjectReference{
+				Type: &kessel.ObjectType{
```

Comment:
not sure of our future use cases - but the name and namespace are hardcoded here , would it be future safe to parametrize them? WDYT?

Reply:
This is a good point that is also causing me a bit of concern. Essentially we are tying the implementation to the specific internal schema we are using. As Alec put it this is accepted technical debt that we don't need to resolve right now.

I see it like you do, parametrizable in the future or maybe plugin driven.

Reply:
For a host repository this should be something like "hbi" namespace and "host" type. We need an "integration repository" for integrations I think, which I think we have: https://github.com/project-kessel/inventory-api/blob/main/internal/data/notificationsintegrations/notificationsintegrations.go

The confusion of hosts vs integrations is my fault – we were originally prioritizing Hosts from HBI in order to get access aware notifications for hosts, but then pivoted for various reasons to start with based RBAC for Notifications integrations as a more minimal starting point that involved less teams.

Reply:
hosts has been changed to rhel_hosts.  Not sure when Chris made that change. However, some of the framework is still "hosts".  When you say "namespace", are you referring a reporter?  HBI is a reporter